src/server/server.rs

```diff
@@ -155,6 +155,9 @@ impl<T: RaftStoreRouter, S: StoreAddrResolver> Server<T, S> {
                 debug!("remove connection token {:?}", token);
                 // if connected to remote store, remove this too.
                 if let Some(store_id) = conn.store_id {
+                    warn!("remove store connection for store {} with token {:?}",
```

Comment:
If the store connection created, we won't close it arbitrarily, so here warn is ok.